apps/console/components/form/formSelect.tsx

```diff
@@ -45,8 +45,8 @@ export const FormSelect = ({
         id={name}
         name={name}
         disabled={readonly}
-        class={`cursor-pointer rounded-sm border outline-0 px-2 pe-6 h-[47px] border-${
```

Comment:
With real talwind, we can't use string templates to generate dynamic class parts within class name, the whole class name must be used.